### DISCUSSION TITLE
Zoom in on images

### DISCUSSION THREAD

**Discussion by stefano799** [sircharlo/meeting-media-manager#1185] [Ideas] [answered]
good morning . thanks for the great work.
is it possible to have the possibility to zoom and move inside the image? it would be useful to move inside the pdf

**Comment by mtdvlpr** [comment, marked as answer]
@stefano799, this feature has already been implemented. You can zoom active images by using your mouse wheel on the image preview and then drag the image preview to show to desired part of the image. Another option is to double click the image preview to switch between different zoom scales.

**Comment by mtdvlpr** [comment]
Hi @stefano799, did you manage to find the feature and is it working as intended?

**Reply by alanfermtz** [reply]
Hello there,
I noticed that is feature is good but inconsistent. Here is an example.

https://user-images.githubusercontent.com/43801184/225174920-1c6eb442-38c9-415d-bee5-ea7b3d172a3d.mp4

This happens with mouse wheel and double click. Also, happens on 2 pc I have and different mouse.

**Reply by sircharlo** [reply]
> Hello there,
> 
> I noticed that is feature is good but inconsistent. Here is an example.
> 
> 
> 
> https://user-images.githubusercontent.com/43801184/225174920-1c6eb442-38c9-415d-bee5-ea7b3d172a3d.mp4
> 
> 
> 
> This happens with mouse wheel and double click. Also, happens on 2 pc I have and different mouse.
> 
> 

I noticed this today too. A work around is to move the image after zooming (even by only one pixel). I'll see if I can see why this is happening.

**Reply by mtdvlpr** [reply]
Moved to #1658
